components/AppBar/examples/AppBarTypicalCollectionViewExample.m

```diff
@@ -0,0 +1,157 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialAppBar.h"
+#import "MaterialAppBar+ColorThemer.h"
+#import "MaterialAppBar+TypographyThemer.h"
+
+@interface AppBarTypicalCollectionViewExample : UICollectionViewController
+
+@property(nonatomic, strong) MDCAppBar *appBar;
+
+@end
+@implementation AppBarTypicalCollectionViewExample
+
+
+- (id)init {
+  UICollectionViewFlowLayout *layout = [[UICollectionViewFlowLayout alloc] init];
+  layout.minimumInteritemSpacing = 10.0f;
+
+  self = [super initWithCollectionViewLayout:layout];
+
+  if (self) {
+    self.title = @"App Bar";
+
+    // Step 2: Initialize the App Bar and add the headerViewController as a child.
+    _appBar = [[MDCAppBar alloc] init];
+    [self addChildViewController:_appBar.headerViewController];
+
+    MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] init];
```

Comment:
please don't set color schemes in init, also please make MDCSemanticColorScheme and MDCTypographyScheme as properties in the interface. see https://github.com/material-components/material-components-ios/pull/3478 for more context

Reply:
Done.